pkgs/by-name/mi/microsocks/package.nix

```diff
@@ -0,0 +1,28 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  lib
+}:
+stdenv.mkDerivation rec {
+  pname = "microsocks";
+  version = "1.0.4";
+  src = fetchFromGitHub {
+    owner = "rofl0r";
+    repo = "microsocks";
+    rev = "v${version}";
+    fetchSubmodules = true;
+    sha256 = "sha256-cB2XMWjoZ1zLAmAfl/nqjdOyBDKZ+xtlEmqsZxjnFn0=";
+  };
+
+  installPhase = ''
+    install -Dm 755 microsocks -t $out/bin/
+  '';
+
+  meta = with lib; {
```

Comment:
Please add the `meta.mainProgram` tag.

Reply:
`microsocks` is default value because is equal to pname.
Why to add `meta.mainProgram`?